ShotgunImportExport/Sources/CSVImportExport.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using UnityEngine;
+using UnityEngine.Timeline;
+using UnityEngine.Playables;
+
+public static class CsvImportExport
+{
+    static readonly int k_FPS = 24;
+    static private readonly string k_directorObjectName = "Sample Shotgun CSV import";
+    static private readonly string k_fileName = "shot.csv";
+    static private readonly string k_pathToFile = Path.Combine("Assets", "CsvImportExport", "Editor");
+    static private readonly string k_timelineObjectName = "Sample Shotgun CSV timeline";
+
+    private static Tuple<ActivationTrack, TimelineClip> CreateActivationTrackAndClip(TimelineAsset timeline, PlayableDirector director)
+    {
+            ActivationTrack track = timeline.CreateTrack<ActivationTrack>(null, "");
+            TimelineClip clip = track.CreateDefaultClip();
+            return new Tuple<ActivationTrack, TimelineClip>(track, clip);
+    }
+
+    [UnityEditor.MenuItem("Shotgun CSV/Import")]
+    public static void ImportCsv ()

```

Comment:
Probably all-caps CSV

Reply:
Style guide states ` Acronyms are PascalCase, unless they are exactly two letters, in which case they are UPPERCASE`

Reply:
Fascinating.